backend/FwLite/LcmCrdt.Tests/Changes/WritingSystemChangeTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LcmCrdt.Changes;
+
+namespace LcmCrdt.Tests.Changes;
+
+public class WritingSystemChangeTests(MiniLcmApiFixture fixture) : IClassFixture<MiniLcmApiFixture>
+{
+    [Fact]
+    public async Task CreatingTheSameWritingSystemShouldWork()
+    {
+        var writingSystem = new WritingSystem()
+        {
+            Id = Guid.NewGuid(),
+            WsId = "de",
+            Abbreviation = "De",
+            Font = "test",
+            Name = "German",
+            Type = WritingSystemType.Analysis
+        };
+        await fixture.Api.CreateWritingSystem(WritingSystemType.Analysis, writingSystem);
+
+        //added via sync
+        await fixture.DataModel.AddChange(Guid.NewGuid(),
+            new CreateWritingSystemChange(writingSystem, WritingSystemType.Analysis, Guid.NewGuid(), 2));
+        var writingSystems = await fixture.Api.GetWritingSystems();
+        writingSystems.Analysis.Should().Contain(ws => ws.WsId == "de");
```

Comment:
It's not clear in this assertion or in the name of the test that it "ShouldWork" by deleting the duplicate.
I think that should be explicit.